kube_helpers.go

```diff
@@ -230,23 +230,23 @@ func createContext() (bool, string) {
 	}
 
 	// connecting to the cluster
-	setCredentialsCmd := ""
+	setCredentialsCmdArgs := []string{}
 	if s.Settings.BearerToken {
 		token := readFile(tokenPath)
 		if s.Settings.Username == "" {
 			s.Settings.Username = "helmsman"
 		}
-		setCredentialsCmd = "kubectl config set-credentials " + s.Settings.Username + " --token=" + token
+		setCredentialsCmdArgs = append(setCredentialsCmdArgs, "config", "set-credentials", s.Settings.Username, "--token="+token)
 	} else {
-		setCredentialsCmd = "kubectl config set-credentials " + s.Settings.Username + " --username=" + s.Settings.Username +
-			" --password=" + s.Settings.Password + " --client-key=" + caKey
+		setCredentialsCmdArgs = append(setCredentialsCmdArgs, "config", "set-credentials", s.Settings.Username, "--username="+s.Settings.Username,
+			"--password="+s.Settings.Password, "--client-key="+caKey)
 		if caClient != "" {
-			setCredentialsCmd = setCredentialsCmd + " --client-certificate=" + caClient
+			setCredentialsCmdArgs = append(setCredentialsCmdArgs, "--client-certificate="+caClient)
```

Comment:
I would add space between `+` for consistency.

Reply:
That is how gofmt formats that kind of line. There is over 120 examples of this kind in the existing code base.